### PR TITLE
"hw3 task4"

### PR DESCRIPTION
**Pull request by E1-Bart0**
Homework 3 task4 Done

### CODE REVIEW

File: test/hw3/test_task4.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+def test_is_armstrong_case2():
+    assert not is_armstrong(10)
+
+
+def test_is_armstrong_case3():
+    assert is_armstrong(9)
+
+
+if __name__ == "__main__":
```
Comment (Cjkjvfnby) [inline test/hw3/test_task4.py]: Why did you need an additional test runner?

File: test/hw3/test_task4.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from All_home_works.hw3.task4 import is_armstrong
+
+
+def test_is_armstrong_case1():
```
Comment (Cjkjvfnby) [inline test/hw3/test_task4.py]: `case1` does not mean anything to me.  Please give a more meaningful name for the test.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

File: test/hw3/test_task4.py (hunk 1 above)
Comment (E1-Bart0) [reply test/hw3/test_task4.py]: This case is in an Example: "
Examples:
- 9 is an Armstrong number, 9 = 9^1 = 9
- 10 is not: 10 != 1^2 + 0^2 = 1
- 153 is : 153 = 1^3 + 5^3 + 3^3 = 1 + 125 + 27 = 153
".
So I decided to check it.

Comment (Cjkjvfnby) [reply test/hw3/test_task4.py]: `if __name__ == "__main__":`  is for executable files, you have have test runner you don't need it.  

Please make sure that there is not such line in the tests of the other tasks.

**Review by Cjkjvfnby — APPROVED** [review]
